### ISSUE TITLE
Don't see option for Preferences > Component Libraries

### ISSUE THREAD

**Issue by QueenLexi13** [ruiqimao/keyboard-pcb-guide#24]
I don't see the option under preferences to add component libraries. Am I missing something?

**Comment by sk8erchen** [comment]
![image](https://user-images.githubusercontent.com/75480055/109248132-39445c00-7820-11eb-8cdf-4a748daa7ac7.png)
Is this what you are looking for? It should look like this, if not, you may need to consider reinstalling.

**Comment by QueenLexi13** [comment]
> ![image](https://user-images.githubusercontent.com/75480055/109248132-39445c00-7820-11eb-8cdf-4a748daa7ac7.png)
> Is this what you are looking for? It should look like this, if not, you may need to consider reinstalling.

Yes, that is what I see. Are one of those the same as "Add Component Libraries"?

**Comment by ghost** [comment]
Note for guy lost in here, after opened `.sch` file then lookup `Preferences > Manage Symbol Libraries` then click folder icon at left-bottom side.

ps. also I think should rename to `Symbol` instead `Component`
